akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/EntityTag.scala

```diff
@@ -37,4 +39,15 @@ object EntityTagRange {
     require(tags.nonEmpty, "tags must not be empty")
     def render[R <: Rendering](r: R): r.type = r ~~ tags
   }
+
+  {
+    // FIXME: Exists only to fix JavaDSL init problem: #19162. Remove in Akka 3.x when "EMPTY" static fields moved
+    val u = Unsafe.instance
+    def initialize(f: java.lang.reflect.Field) =
+      u.putObject(u.staticFieldBase(f), u.staticFieldOffset(f), `*`)
```

Comment:
can be here, or technically inside the 

```
`*`
```

object, but this seemed somewhat safer (earlier init)